hazelcast/src/test/java/com/hazelcast/cache/nearcache/CommonNearCacheTestSupport.java

```diff
@@ -40,20 +40,16 @@ protected NearCacheConfig createNearCacheConfig(String name, InMemoryFormat inMe
     }
 
     protected NearCacheContext createNearCacheContext() {
-        final ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(1);
```

Comment:
This is already done in `createNearCacheExecutor()`, which we call to create the `NearCacheExecutor`.